### ISSUE TITLE
.NET Framework install error (unaffected by --reset flag)

### ISSUE THREAD

**Issue by mus0u** [pauleve/docker-mtgo#130]
running the script (with or without the --reset flag) gives me the error:
```
To run this application, you first must install one of the following versions of the .NET Framework:
 v4.0.30319
Contact your application publisher for instructions about obtaining the appropriate version of the .NET Framework.
```

the output of the script follows, if this is useful:
```
WARNING: You are about to delete /home/musou/.local/share/mtgo and wipe docker volume mtgo-data-musou
Press Enter to continue, CTRL+C to abort

removed '/home/musou/.local/share/mtgo/Xauthority'
removed directory '/home/musou/.local/share/mtgo'
docker volume rm mtgo-data-musou
mtgo-data-musou
docker volume create mtgo-data-musou
mtgo-data-musou
docker run --rm -v mtgo-data-musou:/home/wine/.wine/host panard/mtgo:latest true
docker run --privileged --rm -e DISPLAY -v mtgo-data-musou:/home/wine/.wine/host/ -v mtgo-data-musou:/home/wine/.wine/drive_c/users/ -v /tmp/.X11-unix:/tmp/.X11-unix:rw -v /home/musou/.local/share/mtgo/Xauthority:/home/wine/.Xauthority:ro --net=host --ipc=host -e TZ=/usr/share/zoneinfo/US/Mountain --name mtgo_running panard/mtgo:latest mtgo
winetricks gdiplus=native sound=alsa winegstreamer=disabled wmp=disabled
Executing mkdir -p /home/wine
Using winetricks 20210206-next - sha256sum: c0a29eebfcbec261fcec1cd541f102587c523e03cdb07266c57ead7ca59e8f77 with wine-6.12 and WINEARCH=win32
Using native override for following DLLs: gdiplus
Executing wine regedit C:windowsTemp_nativeoverride-dll.reg
010c:err:vulkan:get_vulkan_driver Wine was built without Vulkan support.
Executing w_do_call sound=alsa
Executing mkdir -p /home/wine
Executing load_sound alsa
Setting sound driver to alsa
Executing wine regedit C:windowsTempset-sound.reg
Using override for following DLLs: winegstreamer
Executing wine regedit C:windowsTemp_disabledoverride-dll.reg
Using override for following DLLs: wmp
Executing wine regedit C:windowsTemp_disabledoverride-dll.reg
wineboot
wine /opt/mtgo/mtgo.exe
01d8:err:winediag:check_version ntlm_auth was not found or is outdated. Make sure that ntlm_auth >= 3.0.25 is in your path. Usually, you can find it in the winbind package of your distribution.
01d8:err:ntlm:ntlm_LsaApInitializePackage no NTLM support, expect problems
01d8:err:ole:apartment_getclassobject DllGetClassObject returned error 0x80131700 for dll L"mscoree.dll"
01d8:err:ole:com_get_class_object no class object {1e66f26b-79ee-11d2-8710-00c04f79ed0d} could be created for context 0x1
0208:err:ole:apartment_getclassobject DllGetClassObject returned error 0x80131700 for dll L"mscoree.dll"
0208:err:ole:com_get_class_object no class object {1e66f26b-79ee-11d2-8710-00c04f79ed0d} could be created for context 0x1
0208:err:ole:CoReleaseMarshalData StdMarshal ReleaseMarshalData failed with error 0x8001011d
```

i'm running on Debian 10.10 (buster) if that makes a difference. i tried building my own version of the mtgo image from the Dockerfile in this repository with `dotnet40` instead of `dotnet46` to see if that would fix the problem, but it looks like the base image the Dockerfile uses isn't publicly fetchable, so i'm unable to see if that resolves the issue.

**Comment by kenny-designs** [comment]
Hey! I'm also running Debian 10.10 and was just trying to set up docker-mtgo for the first time but was experiencing some .NET related issues as well. The --reset flag didn't seem to help either. However, I tried running `./run-mtgo --update` and that appears to have done the trick.  Everything downloaded just fine and I was able to log in and successfully do a goldfish test with my deck. It's not much but I hope that's enough to get it workin' for you too!
